script/build

```diff
@@ -21,11 +17,16 @@ const argv = yargs
   .describe('create-debian-package', 'Create .deb package (Linux only)')
   .describe('create-rpm-package', 'Create .rpm package (Linux only)')
   .describe('compress-artifacts', 'Compress Atom binaries (and symbols on macOS)')
+  .describe('generate-api-docs', 'Only build the API documentation')
   .describe('install', 'Install Atom')
   .string('install')
   .wrap(yargs.terminalWidth())
   .argv
 
+// Needed so we can require src/module-cache.coffee during generateModuleCache
```

Comment:
Moving this just because it isn't used in the previous section or was there another benefit?

Reply:
I think this is a leftover from a separate change where I was trying to get rid of _all_ the dependency `require`s (to exit early when `--help` is called), however as the arguments are parsed _in_ a dependency, it doesn't actually make that much of a difference. I can remove this change if you would prefer.

Reply:
Removing it would be ideal, thanks a bunch!